### ISSUE TITLE
hide playlist wich mouse cursor

### ISSUE THREAD

**Issue by region51** [gnome-mpv/gnome-mpv#176]
![screenshot from 2016-05-25 16-37-21](https://cloud.githubusercontent.com/assets/13280183/15541977/78b054f4-2297-11e6-8222-0255293e5950.png)
10px?

**Comment by gnome-mpv** [comment]
This is intentional. It'd be difficult to click the playlist's resize handle if it's allowed to become too small. You can hide the playlist by clicking "Toggle Playlist" in the menu button.

**Comment by region51** [comment]
>You can hide the playlist by clicking "Toggle Playlist" in the menu button.

know it all.

>This is intentional. It'd be difficult to click the playlist's resize handle if it's allowed to become too small.

It's a pity! I don't speak English.
In Baka-mplayer https://github.com/u8sand/Baka-MPlayer
this is done properly

**Comment by region51** [comment]
@ElvisPresly привет, у тебя как с английским? Если я напишу по-русски переведешь?

**Comment by gnome-mpv** [comment]
Okay. So basically, the playlist should be hidden if the handle is dragged past the minimum size?

**Comment by TingPing** [comment]
I agree with @gnome-mpv, dragging to hide is very non-obvious behavior and has been a UI paradigm to avoid for decades now. It happens on accident and users never figure it out.

**Comment by region51** [comment]
@gnome-mpv example video
https://www.dropbox.com/s/5xdfpwrzsf139id/baka.mkv?dl=0

**Comment by ElvisPresly** [comment]
@region51 привет, вообще средненько, длинные тексты переводить нет желания, если только интерфейс программ, типа кнопка сюда, кнопка туда) А что хотел предложить перевести?

**Comment by region51** [comment]
@ElvisPresly да уже не надо, что я хотел сказать, видно на видео.

**Comment by gnome-mpv** [comment]
I guess I can implement that as a non-default option.
